.gitignore

```diff
@@ -41,4 +41,5 @@ cmake_install.cmake
 /visual_studio/tmp/
 /visual_studio/Debug/
 /visual_studio/.vs/
+visual_studio/a1.vcxproj.user
```

Comment:
does this need a slash at the front like rest of the `visual_studio` stuff?

Reply:
Oh whoops yeah. Good call.